content/en/sensitive_data_scanner/_index.md

```diff
@@ -0,0 +1,194 @@
+    - Arbitrary zero-width assertions
+    - Subroutine references and recursive patterns
+    - Conditional patterns
+    - Backtracking control verbs
+    - The \C "single-byte" directive (which breaks UTF-8 sequences)
+    - The \R newline match
+    - The \K start of match reset directive
+    - Callouts and embedded code
+    - Atomic grouping and possessive quantifiers
+1. In the **Create keyword dictionary** section, add keywords to refine detection accuracy when matching regex conditions. For example, if you are scanning for a sixteen-digit Visa credit card number, you can add keywords like `visa`, `credit`, and `card`. You can also require that these keywords must be within a specified number of characters of a match. By default, keywords must be within 30 characters before a matched value.
+1. For **Define actions on match**, select the action you want to take for the matched information.
+    - **Redact**: Replaces all matching values with the text you specify in the **Replacement text** field.
+    - **Partially Redact**: Replaces a specified portion of all matched data. In the **Redact** section, specify the number of characters you want to redact and which part of the matched data to redact.
+    - **Hash**: Replaces all matched data with a unique identifier. The UTF-8 bytes of the match is hashed with the 64-bit fingerprint of farmhash.
+    - **Note**: Redaction, partial redaction, and hashing are all irreversible actions.
+1. Optionally, add tags you want to associate with events where the values match the specified regex pattern. Datadog recommends using `sensitive_data` and `sensitive_data_category` tags. These tags can then be used in searches, dashboards, and monitors. See [Control access to events with sensitive data](#control-access-to-events-with-sensitive-data) for information on how to use tags to determine who can access events containing sensitive data.
+1. For **Set priority level**, select the priority level for the rule based on your business needs.
+1. In the **Name and describe the scanning rule** section, enter a name for the rule. Optionally, add a description.
+1. Click **Add Rule**.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+**Notes**:
+
+- Any rules that you add or update only affect data coming into Datadog after the rule was defined.
+- Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.
+- To turn off Sensitive Data Scanner entirely, set the toggle to off for each Scanning Group and Scanning Rule so that they are disabled.
+
+See [Investigate Sensitive Data Issues][5] for details on how to use the [Summary][6] page to triage your sensitive data issues.
+
+### Control access to events with sensitive data
+
+To control who can access events containing sensitive data, use tags added by the Sensitive Data Scanner to build queries with role-based access control (RBAC). You can restrict access to specific individuals or teams until the data ages out after the retention period.
```

Comment:
Is this specifically talking about logs? What about APM and RUM?
Should we link to [How to Set Up RBAC for Logs](https://docs.datadoghq.com/logs/guide/logs-rbac/?tab=ui)?

Reply:
Yes, this is specifically about logs for now, and I think it would be good to link that guide.